examples/notebook_tutorials/new_test.py

```diff
@@ -0,0 +1,57 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
You missed removing this file: . See https://github.com/quantumlib/Cirq/pull/2657#issuecomment-579439362